third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.h

```diff
@@ -38,6 +43,30 @@ namespace google_smart_card {
 // already been initialized.
 void InitializeAndRunPcscLiteServer();
```

Comment:
The global declaration should be removed, and its comment now should be placed right before the corresponding method declaration.